C/tests/c4QueryTest.cc

```diff
@@ -464,6 +470,41 @@ N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query FTS", "[Query][C][FTS]") {
     c4slice_free(matched);
 }
 
+N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query FTS (Partial)", "[Query][C][FTS]") {
```

Comment:
I have seen the other test above was changed to test both JSON and SQL++. Would it be worth to do that here too?

Reply:
I can do it